microservices.yml

```diff
@@ -133,6 +133,17 @@ commands:
           command: |
             make deploy
 
+  run-dependent-service-dns-tests:
+    description: Test dependent service DNS availability
+    steps:
+      - run:
+          name: Ping dependent service hostnames to check if DNS is reachable
+          command: |
+            ping eaiws-qa.mgmresorts.local -c 5
```

Comment:
i'm afraid that this host has ICMP disabled (or maybe it is not working right now)

Reply:
Good catch. I'll update to use `host -t a {domain}` command instead.
Will verify before I make the change.